spec/requests/api/moves_controller_update_spec.rb

```diff
@@ -150,10 +150,10 @@
             }
           end
 
-          it 'updates the moves documents' do
-            expect(move.reload.documents).to match_array(before_documents)
+          it 'updates the moves profile documents' do
```

Comment:
should we call it just "profile documents" instead of "moves profile documents"

Reply:
I guess move is the resource under test so I felt it made sense to reference it. Happy to change if you feel strongly.